Code/Tools/ProjectManager/Source/ProjectsScreen.cpp

```diff
@@ -924,6 +1025,34 @@ namespace O3DE::ProjectManager
         return displayFirstTimeContent;
     }
 
+    bool ProjectsScreen::StartProjectExport(const ProjectInfo& projectInfo, bool skipDialogBox)
+    {
+        bool proceedToExport = skipDialogBox;
+        if (!proceedToExport)
+        {
+            QMessageBox::StandardButton buildProject = QMessageBox::information(
+                this,
+                tr("Exporting \"%1\"").arg(projectInfo.GetProjectDisplayName()),
+                tr("Ready to export \"%1\"?").arg(projectInfo.GetProjectDisplayName()),
```

Comment:
Do we want to add a message asking the user to ensure that you have "Configured the settings" using the appropriate menu option

Reply:
I'm wondering if we can add a settings button at this prompt that also launches the settings panel.
 Worst case, we could just add a simple text reminder reminding them to check the configuration settings for this PR